api/v3/Job/Monerisvaultrecurringcontributions.php

```diff
@@ -104,7 +104,7 @@ function civicrm_api3_job_Monerisvaultrecurringcontributions($params) {
 SELECT
   cr.id as recur_id, cr.contact_id, cr.payment_token_id,
   c.id as original_contribution_id, c.contribution_status_id,
-  c.total_amount, c.currency, c.invoice_id,
+  c.amount, c.currency, c.invoice_id,
```

Comment:
@monishdeb doesn't this need to be cr table alias?

Reply:
Thanks @seamuslee001 for pointing this. I have updated the patch now.